api-gateway/caching-api-response/modules/health-check/src/main/java/com/project/example/healthcheck/HealthcheckHandlerFactory.java

```diff
@@ -0,0 +1,47 @@
+package com.project.example.healthcheck;
+
+import io.knotx.server.api.handler.RoutingHandlerFactory;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.healthchecks.Status;
+import io.vertx.reactivex.core.Vertx;
+import io.vertx.reactivex.ext.healthchecks.HealthCheckHandler;
+import io.vertx.reactivex.ext.healthchecks.HealthChecks;
+import io.vertx.reactivex.ext.web.RoutingContext;
+import io.vertx.reactivex.ext.web.client.WebClient;
+
+public class HealthcheckHandlerFactory implements RoutingHandlerFactory {
+
+  @Override
+  public String getName() {
+    return "healthcheck";
+  }
+
+  @Override
+  public Handler<RoutingContext> create(Vertx vertx, JsonObject config) {
+    HealthChecks checks = HealthChecks.create(vertx);
+    checks.register("API check", 200, future -> {
```

Comment:
We expose API proxy, however, it can not work without Target API. Target API is deployed by Docker Swarm, so we are not able to validate it during image building. Can you please remove all checks and return always OK response?

Reply:
fixed